stable/artifactory-ha/templates/nginx-deployment.yaml

```diff
@@ -29,6 +29,17 @@ spec:
       imagePullSecrets:
       - name: {{ .Values.imagePullSecrets }}
     {{- end }}
+      initContainers:
```

Comment:
This initContainer needs to be wrapped with `{{- if .Values.nginx.persistence.enabled }}`